Classes/DDFileLogger.m

```diff
@@ -433,6 +434,28 @@ - (NSString *)newLogFileName {
     return [NSString stringWithFormat:@"%@ %@.log", appName, formattedDate];
 }
 
+- (void)modifyLogFileCreateDateIfNeed:(NSString *)filePath
+{
+    NSDate* fileCreateDate = [NSDate date];
+    if (_lastLogFileCreateDate == nil) {
+        DDLogFileInfo *logFileInfo = [self sortedLogFileInfos].firstObject;
+        if (logFileInfo) {
+            _lastLogFileCreateDate = logFileInfo.creationDate;
+        }
+    }
+    if (_lastLogFileCreateDate && (int)_lastLogFileCreateDate.timeIntervalSince1970 >= (int)fileCreateDate.timeIntervalSince1970) {
+        fileCreateDate = [_lastLogFileCreateDate dateByAddingTimeInterval:1];
```

Comment:
Thanks rivera-ernesto prompt reply，but this situation's real place in our projects, my example in order to confirm and amplify the emergence of possible.
when log files rapid rolled, 1 second generates two files are very possible, file creation time is very likely the same.
Please solve this problem, makes the code more robust.
You can not assume that the user how to use your library.